src/main/java/org/jenkinsci/test/acceptance/po/Parameter.java

```diff
@@ -2,20 +2,30 @@
 
 import org.openqa.selenium.By;
 
+import java.util.LinkedList;
+import java.util.List;
+
 /**
  * Parameter for builds.
  * <p/>
- * Use {@link Describable} annotation to register an implementation.
+ * Use {@link Describable} annotation to register an implementation. The value of {@link Describable} must be the
+ * corresponding caption of the add parameter menu item AND the corresponding {@link hudson.model.ParameterDefinition}
+ * class. For example {"String Parameter", "hudson.model.StringParameterDefinition"}
```

Comment:
You have preserved the text description to kep it working in `Job#addParameter()`, right? Please add TODO get rid of that.